README.md

```diff
@@ -49,23 +49,38 @@ More details on contributing to the code are in the [CONTRIBUTING.md](CONTRIBUTI
 
 # Docker for local development
 For a standalone ohm-website development server, follow the steps below:
-1. Create a ohm-docker.env from ohm-docker.env.example. If you are using the database as part the docker setup in this repo, leave the defaults. This command is an easy way to do that:
+
+1. Check out this repository to your local machine. Make sure you also have Docker installed. You can get Docker at https://docs.docker.com/get-docker/
+
+2. Make a new file at `config/storage.yml` with the following contents:
```

Comment:
@danrademacher this should just be creating a copy of `example.storage.yml` https://github.com/OpenHistoricalMap/ohm-website/blob/staging/config/example.storage.yml — if it helps we can create these local files as part of the docker build.

Reply:
I updated the README to suggest a `cp` of that file into the new file, similar to the other steps.